frontend/helpers/tillDuration.ts

```diff
@@ -20,11 +20,12 @@ export const formatSeconds = function (seconds: number): string {
     return formatDuration(duration);
 };
 
-export const formatInterval = function (startDate: Date, endDate: Date): string {
-    if (startDate < endDate) {
+export const formatInterval = function (startDate: Date, endDate: Date, isCountUp: boolean = false): string {
+    if (startDate < endDate || isCountUp) {
+        // switch startDate and endDate to act as a count-up timer
```

Comment:
Do we actually use `formatDistance` format anywhere else then? Otherwise I would suggest to completely remove `formatDistance` hack from here completely and use `0h 0m 0s` format everywhere